apps/sptribs/preview/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../../base
+  - ../../identity/identity.yaml
+namespace: sptribs
+patchesStrategicMerge:
```

Comment:
You may just want an environmental patch override, but not required.
like this
https://github.com/hmcts/cnp-flux-config/blob/master/apps/et/et-cos/aat.yaml
https://github.com/hmcts/cnp-flux-config/blob/6113a4ce2d227d245c5193aa1981eec6a836448b/apps/et/aat/base/kustomization.yaml#L10